tests/benchmark/spaces_benchmarks/bm_spaces_int8.cpp

```diff
@@ -46,11 +46,12 @@ bool avx512_f_bw_vl_vnni_supported = opt.avx512f && opt.avx512bw && opt.avx512vl
 INITIALIZE_BENCHMARKS_SET_L2_IP(BM_VecSimSpaces_Integers_INT8, INT8, AVX512F_BW_VL_VNNI, 32,
                                 avx512_f_bw_vl_vnni_supported);
 INITIALIZE_BENCHMARKS_SET_Cosine(BM_VecSimSpaces_Integers_INT8, INT8, AVX512F_BW_VL_VNNI, 32,
```

Comment:
The `;` is redundant no?

Reply:
I think so, but it messed with the clang-format so I added it